Frameworks/CoreFoundation/Base.subproj/CFSystemDirectories.c

```diff
@@ -55,7 +55,8 @@ CFSearchPathEnumerationState __CFGetNextSearchPathEnumeration(CFSearchPathEnumer
 #endif
 
 
-#if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_WINDOWS
+// WINOBJC: Disable this function only for WinObjC as we don't provide system intrinsics for it.
```

Comment:
> system intrinsics [](start = 70, length = 18)

I'm not seeing the system intrinsics here. what are we missing to allow this? Who are the callers of this? do we need to adjust a .def?